recipes/perfplot/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "perfplot" %}
+{% set version = "0.9.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/perfplot-{{ version }}.tar.gz
+  sha256: b38bbd402f6bb90c10e6a44dc2573039db3793ced98a1dfde0677dbff37262a2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - dufte >=0.2.20
```

Comment:
I am afraid we don't have ``dufte`` on ``conda-forge``, you will need to add the recipe for that

Reply:
Already done in https://github.com/conda-forge/staged-recipes/pull/15226, just waiting for it to show up in the repo!